assets/i18n/qqq.json

```diff
@@ -209,8 +209,8 @@
 	"I18N_LEARNER_DASHBOARD_EMPTY_COMPLETED_EXPLORATIONS": "Text displayed in the learner dashboard. - This text is shown when there are no explorations in the completed section of the user.",
 	"I18N_LEARNER_DASHBOARD_EMPTY_EXPLORATION_PLAYLIST": "Text displayed in the learner dashboard. - This text is shown when there are no explorations in the playlist of the user.",
 	"I18N_LEARNER_DASHBOARD_EMPTY_FEEDBACK_THREADS": "Text displayed in the learner dashboard. - This text is shown when there are no feedback threads to which the user is subscribed.",
-	"I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_COLLECTIONS": "Text displayed in the learner dashboard. - This text is shown when there are no collections in the incomplete section of the user.",
-	"I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_EXPLORATIONS": "Text displayed in the learner dashboard. - This text is shown when there are no explorations in the incomplete section of the user.",
+	"I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_COLLECTIONS": "Text displayed in the learner dashboard. - This text is shown when there is one or more than one collections in the complete section but no collections in the incomplete section of the user.",
```

Comment:
... when there is at least one collection in the complete section ...

Reply:
Done!